src/logic/addressBook/store/actions/addAddressBookEntry.ts

```diff
@@ -1,7 +1,12 @@
 import { createAction } from 'redux-actions'
+import { AddressBookEntry } from 'src/logic/addressBook/model/addressBook'
 
 export const ADD_ENTRY = 'ADD_ENTRY'
 
-export const addAddressBookEntry = createAction(ADD_ENTRY, (entry) => ({
-  entry,
-}))
+export const addAddressBookEntry = createAction(
+  ADD_ENTRY,
+  (entry: AddressBookEntry, shouldAvoidUpdatesNotifications?: boolean) => ({
```

Comment:
I thought it wouldn't be necessary, but while reviewing the code it was hard for me to remember the meaning of boolean passed as a second argument to `addAddressBookEntry`

That being said, can we make the `shouldAvoidUpdatesNotifications` as part of an options object?

Something around:
```typescript
(entry: AddressBookEntry, { notifyEntryUpdate }: Options = { notifyEntryUpdate = true }) => // ...
```

Yes, also changed the flag name. Everything is a suggestion.